guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -7,6 +7,8 @@ You can use the Leapp tool to upgrade as well as to help detect and resolve issu
 ifdef::satellite[]
 * Review Known Issues before you begin an upgrade.
 For more information, see {ReleaseNotesDocURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+* Review Known Issues for Leapp before you begin an upgrade.
```

Comment:
```suggestion
* Review known issues before you begin an upgrade.
```
I don't think you need "for Leapp" because the context and link make it clear.
I think "known issues" should be lower case because it looks weird in caps.

Reply:
@apinnick so the step just before this said to refer to "Known Issues", referring to the RNs.
I have combined the two steps now.